MeNext/Layout/MainPage.cs

```diff
@@ -63,7 +65,7 @@ public void SomethingChanged()
                     foreach (var page in this.pages) {
                         this.Children.Add(page);
                     }
-                } else {
+                } else { // No Event no tabs.
```

Comment:
Put this comment on the next line. I realise we have no style guide but, if we did, it would say that comments should typically be on their own line